development.ini.example

```diff
@@ -131,17 +151,17 @@ base_address = http://localhost:6543/
 ## Supported update types
 update_types = bugfix enhancement security newpackage
 
-## Supported architechures
-##
-## To handle arch name changes between releases, you
-## can also configure bodhi to support one arch *or*
-## another. For example, EPEL5 mashes produce 'ppc'
-## repos, where EPEL6 produces 'ppc64'. To handle this
-## scenario, you can specify something like:
-##
-##   arches = ppc/ppc64
+
+## Primary architechures by release
 ##
-arches = x86_64 i386 armhfp
+## {release}_{version}_primary_arches: Releases that have alternative arches must define their
+##      primary arches here. Any arches found during mashing that are not present here are asssumed
+##      to be alternative arches. This is used during the wait_for_repo() step of the mash where
+##      Bodhi polls the master repo to find out whether the mash has made it to the repo or not.
+##      Bodhi looks for primary arches with the {release}_{request}_master_repomd setting above, and
+##      for alternative arches at the {release}_{request}_alt_master_repomd setting above. If this
+##      is not set, Bodhi will assume the release only has primary arches.
```

Comment:
I was going for a bit of a docblock style here, so I think I prefer it on this line as is.